.github/workflows/beam_LoadTests_Java_CoGBK_Dataflow_Batch_Java.yml

```
@@ -0,0 +1,133 @@
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: LoadTests Java CoGBK Dataflow Batch Java
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '0 10 * * *'
+  workflow_dispatch:
+
+# Setting explicit permissions for the action to avoid the default permissions which are `write-all` in case of pull_request_target event
+permissions:
+  actions: write
+  pull-requests: read
+  checks: read
+  contents: read
+  deployments: read
+  id-token: none
+  issues: read
+  discussions: read
+  packages: read
+  pages: read
+  repository-projects: read
+  security-events: read
+  statuses: read
+
+# This allows a subsequently queued workflow run to interrupt previous runs
+concurrency:
+  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.sha || github.head_ref || github.ref }}-${{ github.event.schedule || github.event.comment.body || github.event.sender.login }}'
+  cancel-in-progress: true
+
+env:
+  GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
+  GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GE_CACHE_USERNAME }}
+  GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GE_CACHE_PASSWORD }}
+
+jobs:
+  beam_LoadTests_Java_CoGBK_Dataflow_V2_Batch_Java:
+    if: |
+      github.event_name == 'workflow_dispatch' ||
+      github.event_name == 'schedule' ||
+      (contains(github.event.comment.body, 'Run Load Tests Java') &&
+         contains(github.event.comment.body, 'CoGBK Dataflow V2 Batch'))
+    runs-on: [self-hosted, ubuntu-20.04, main]
+    timeout-minutes: 720
+    name: ${{matrix.job_name}} (${{matrix.job_phrase_1}} ${{matrix.java_version}} ${{matrix.job_phrase_2}})
+    strategy:
+      fail-fast: false
+      matrix:
+        job_name: ["beam_LoadTests_Java_CoGBK_Dataflow_V2_Batch_Java"]
+        job_phrase_1: ["Run Load Tests Java"]
+        job_phrase_2: ["CoGBK Dataflow V2 Batch"]
+        java_version: ['11','17']
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup repository
+        uses: ./.github/actions/setup-action
+        with:
+          comment_phrase: ${{ matrix.job_phrase_1 }} ${{matrix.java_version}} ${{ matrix.job_phrase_2 }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_job: ${{ matrix.job_name }} ${{ matrix.job_phrase_1 }} ${{matrix.java_version}} ${{ matrix.job_phrase_2 }}
+      - name: Set up Java${{ matrix.java_version }}
+        uses: actions/setup-java@v3.8.0
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.java_version }}
```

```suggestion
      - name: Setup environment
        uses: ./.github/actions/setup-environment-action
        with:
          java-version: ${{ matrix.java_version }}
```
Please apply it for all workflow files in this PR